script_vts.py

```diff
@@ -87,9 +86,9 @@ def file_rep_api_req(key, hash):
 
     if response.status_code == 200:
         return response.text
-    else:
-        print(f"[-] HTTP response wasn't successful: {response.status_code}")
-        return
+
+    print(f"[-] HTTP response wasn't successful: {response.status_code}")
+    return
 
 
 def parse_json(res, string, key):
```

Comment:
parse_json does a lot more than parse JSON. We'll need to split out a lot of the functionality in future versions.